examples/gno.land/r/demo/tamagotchi/z0_filetest.gno

```diff
@@ -0,0 +1,65 @@
+package main
+
+import (
+	"std"
+	"time"
+
+	"gno.land/r/demo/tamagotchi"
+)
+
+func main() {
+	t := tamagotchi.New("Gnome")
+
+	println("-- INITIAL")
+	println(t.Markdown())
+
+	// println("-- WAIT 20 minutes")
+	// println(t.Markdown())
```

Comment:
You can actually "sleep" right now, it's just a bit hidden. See `examples/gno.land/p/demo/flow/io_test.gno` -- it's `internal/os_test.Sleep`.

When we sort out and fix the test functions for sleeping, the PR can change the code here -- as it would have to do it for p/demo/flow anyway :)